modules/ROOT/pages/asset-details.adoc

```diff
@@ -4,9 +4,31 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
+== View Assets
+
+To view assets:
+
+. From the public https://www.anypoint.mulesoft.com/exchange/[Exchange], click an asset and view its description. See the Explore Asset Information section for more information about viewing an asset's detail.
+. View the documentation and metadata to learn more about the asset.
+. Click *Login* to view assets published into Exchange by your organization.
+. After you log in, click *Public Portal* in your organization to view public APIs available to anyone.
+
+After you log in to Exchange, you can create assets.
+
+== Test APIs by Using the Mocking Service
+
+To use the API mocking service for an asset:
+
+. Click *Any Types* > *REST APIs* and choose any API.
+. Click the asset and view the API functions in the left navigation area.
+. Click a function to show more information about that function.
+. At the right side of the screen, test the API by entering sample values to send to the API and get responses from the API.
+
+== Explore Asset Information
```

Comment:
I have the same question as your SME reviewer: how is this different from the goal of View Assets?
And why is testing assets included in the container "Asset Details"?

Could it be that you really want an org like this:

 = Exploring Assets (all level-1 titles should use gerunds: see our templates)
 text about why audience wants/needs to do this: advantages
  == View Asset Information (all topics with steps need an imperative verb; see templates)
  intro to task, if any: maybe about info you see before and after logging in?
  steps, ending with clicking an asset
  what you see displayed after clicking an asset
 == View Assets by Business Group....
 == View REST API....
 
 Followed by a separate container...

  = Testing Assets (gerund)
  blah blah blah as above
   == Test Assets by Using the API Mocking System (imperative)
 blah blah blah steps
  == Test [Assets or API?] by Using API Notebook (imperative)
 blah blah blah steps